RLBot/Manager/Match.cs

```diff
@@ -143,11 +143,12 @@ private void EnsureGameConnection()
     public void SetGameState(
         Dictionary<int, DesiredBallStateT>? balls = null,
         Dictionary<int, DesiredCarStateT>? cars = null,
-        DesiredGameInfoStateT? gameInfo = null,
+        Dictionary<int, DesiredBoostStateT>? boostPads = null,
```

Comment:
Boost pads aren't included in SetGameState because RLBot can't set the state of boost pads. If anything, it should be removed from the flatbuffers schema.

Reply:
Removed in 67a62c85ab93e3c8fad0bec58a37d0e2e640658a (https://github.com/RLBot/flatbuffers-schema/commit/9fed2141187c2f7b48a3d1a4b3412f54f4e95380)